### PR TITLE
office-addin-manifest: add 'validate' command to README.md

### PR DESCRIPTION
**Pull request by akrantz**

### CODE REVIEW

**Comment by akrantz** [comment]
@kbrandl Please review. Glad to make any adjustments to the readme text. Thanks.

File: packages/office-addin-manifest/README.md [hunk 1]
```diff
@@ -52,3 +53,13 @@ This value is the `<DisplayName>` element of `<OfficeApp>`.
 For more info, see [OfficeApp documentation](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/manifest/officeapp).
 
 #
+
+### validate 
+Determines whether the Office Add-in manifest is valid.
+
+Syntax:
+
+`office addin-manifest validate <manifest> [options]`
```
Comment (TCourtneyOwen) [inline packages/office-addin-manifest/README.md]: do we need options here?

Comment (akrantz) [reply packages/office-addin-manifest/README.md]: Nope.

**Comment by akrantz** [comment]
Found a bug related to the "chalk" dependency:
```
c:\dev\git\Office-Addin-Scripts>npx office-addin-manifest validate ..\Office-Addin-TaskPane\manifest.xml
npx: installed 12 in 2.783s
Cannot find module 'chalk'
```

**Comment by akrantz** [comment]
Pushed a fix.

File: packages/office-addin-manifest/README.md [hunk 2]
```diff
@@ -52,3 +53,13 @@ This value is the `<DisplayName>` element of `<OfficeApp>`.
 For more info, see [OfficeApp documentation](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/manifest/officeapp).
 
 #
+
+### validate 
+Determines whether the Office Add-in manifest is valid.
+
+Syntax:
+
+`office addin-manifest validate <manifest>`
```
Comment (kbrandl) [inline packages/office-addin-manifest/README.md]: @akrantz Is this command missing the hyphen after `office` (in `office addin-manifest`)?  If yes, then please fix the same typo in the other two commands in the README (for `info` and `modify`).

Comment (akrantz) [reply packages/office-addin-manifest/README.md]: yes, fixed.

**Pull request merged**